app/views/hosts/_templates.html.erb

```diff
@@ -0,0 +1,38 @@
+<% if @host.managed? %>
```

Comment:
This should be in the controller, before attempting to find all the templates for an unmanaged host.
You are also missing the `SETTINGS[:unattended]` part.